src/util/validation.js

```diff
@@ -7,5 +7,8 @@ export const querySchema = Joi.object({
 	mockResponse: Joi.object(),
 	params: Joi.object(),
 	type: Joi.string(),
+	meta: Joi.object({
+		method: Joi.string().only('get', 'post', 'delete', 'patch'),
```

Comment:
`Joi.string().lowercase().only('get', ...)` to support any case seems nice to me. i could imagine getting an error and not realizing it's because my method is supposed to be lowercase.